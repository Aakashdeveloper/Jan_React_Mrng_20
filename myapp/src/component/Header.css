header{
    background:olive
}

.logo{
    color:#fff;
    font-size:40px;
    text-align:center
}

img {
    font-size:40px;
}