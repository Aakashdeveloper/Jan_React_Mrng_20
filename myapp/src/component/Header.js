import React from 'react';

const Header = () => {
    return(
        <div>
            <center>
                <h2>React App</h2>
            </center>
            <hr/>
        </div>
    )
}

export default Header;