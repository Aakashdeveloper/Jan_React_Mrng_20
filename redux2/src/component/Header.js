import React from 'react';
//import { Link } from 'react-router-dom';

const Header = () => {
    return(
        <header>
            <div>
                <h1>My Header</h1>
            </div>
        </header>
    )
}

export default Header;