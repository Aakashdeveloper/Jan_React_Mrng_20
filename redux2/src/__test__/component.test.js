import React from 'react';
import Header from '../component/Header';
import {create} from 'react-test-renderer';

describe('Snapshot test', () => {
    test('testing my header component',() => {
        let tree = create(<Header/>)
        expect(tree.toJSON()).toMatchSnapshot()
    })
    
})  