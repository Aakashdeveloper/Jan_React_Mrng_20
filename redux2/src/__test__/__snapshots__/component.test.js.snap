// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test testing my header component 1`] = `
<header>
  <div>
    <h1>
      My Header
    </h1>
  </div>
</header>
`;
